src/main/scala/com/databricks/spark/avro/SchemaConverters.scala

```diff
@@ -152,6 +154,13 @@ object SchemaConverters {
         case (IntegerType, INT) | (BooleanType, BOOLEAN) | (DoubleType, DOUBLE) |
              (FloatType, FLOAT) | (LongType, LONG) =>
           identity
+        case (TimestampType, LONG) =>
+          (item: AnyRef) => {
```

Comment:
I think we should remove the brace, and make it one line

Reply:
Ok.